File: tests/expected/global.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

final int code_0 = 0;
final int code_1 = 1;
final List<int> l_a = [code_0, code_1];
final String code_a = "a";
final String code_b = "b";
final List<String> l_b = [code_a, code_b];
main(List<String> argv) {
  for (final i in l_a) {
    print(sprintf("%s", [i]));
  }
  for (final j in l_b) {
    print(sprintf("%s", [j]));
  }

  if (["a", "b"].contains("a")) {
    print(sprintf("%s", ["OK"]));
  }
}

File: tests/expected/global.go
package main

import (
	"fmt"
	"github.com/electrious/refutil"
)

var Code0 int = 0
var Code1 int = 1
var LA []int = []int{Code0, Code1}
var CodeA string = "a"
var CodeB string = "b"
var LB []string = []string{CodeA, CodeB}

func main() {
	for _, i := range LA {
		fmt.Printf("%v\n", i)
	}
	for _, j := range LB {
		fmt.Printf("%v\n", j)
	}
	if refutil.Contains([]string{"a", "b"}, "a") {
		fmt.Printf("%v\n", "OK")
	}
}
